assets/dev/js/editor/views/base-container.js

```diff
@@ -93,25 +93,28 @@ module.exports = Marionette.CompositeView.extend( {
 	},
 
 	createElementFromModel( model, options = {} ) {
+		let container = this.getContainer();
+
 		if ( model instanceof Backbone.Model ) {
 			model = model.toJSON();
 		}
 
+		if ( elementor.helpers.maybeDisableWidget( model.widgetType ) ) {
+			return;
+		}
+
 		model = Object.assign( model, model.custom );
 
-		if ( 'section' === model.elType ) {
+		// When a section is about to be created inside an existing element, it's converted to an inner-section, unless
+		// it's placed in the document level (for example, from a JSON template).
```

Comment:
Try to use `isValidChild` otherwise add `container` and a `todo` comment

Reply:
Done.